src/operations/threshold.ts

```diff
@@ -70,18 +74,20 @@ export type ThresholdOptions =
  * Compute threshold value for an image using the specified algorithm.
  * @param image - The grey image.
  * @param algorithm - Algorithm that defines the threshold.
+ * @param slots - The number of slots that histogram can have.
  * @returns The threshold value for the image.
  */
 export function computeThreshold(
   image: Image,
   algorithm: ThresholdAlgorithm = 'otsu',
+  slots?: number,
```

Comment:
@stropitek @targos : We have now 2 optional arguments ... I would make a breaking change to have the function:
```js
export function computeThreshold(image, options = {})
const {algorithm = 'otsu', slots} = options;
```

Reply:
sgtm